.github/workflows/ci-integration-tests.yaml

```diff
@@ -26,6 +26,13 @@ jobs:
       TF_VAR_sysdig_secure_api_token: ${{secrets.KUBELAB_SECURE_API_TOKEN}}
 
     steps:
+      # remove this tf setup with 1.3.1 is released for github runner (it fixes an impacting bug)
+      # https://github.com/actions/runner-images/pull/6346
+      # hashicorp/setup-terraform@v2 requires wrapper set to false
+      # https://github.com/newcontext-oss/kitchen-terraform/issues/411#issuecomment-953920206
+      - uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_wrapper: false
```

Comment:
temporal hack until terraform 1.3.1 is released in gh runner.
terraform 1.3.0 does not work correctly on kitchen destroy phase